### DISCUSSION TITLE
Dell P2417H Not Supported?

### DISCUSSION THREAD

**Discussion by AustinBMedia** [xanderfrangos/twinkle-tray#358] [Monitor Compatibility Issues]
Hi,

I updated my Twinkle Tray app (received through Microsoft Store), and now, my monitor (hooked directly into my laptop via a Rocketfish Premium High Speed HDMI cable), and now, even though I have DDC/CI enabled in my monitor settings, Twinkle Tray no longer detects my monitor.

Attached is a screenshot of the Twinkle Tray monitor settings menu.

![image](https://user-images.githubusercontent.com/104846014/166475775-99806d86-cb97-4237-bc81-a0ec2332e187.png)

**Comment by vEonz121** [comment]
Same with Acer R221Q. Stopped working after an update. On reinstall the issue remains.

I followed the troubleshooting guide written on here: https://github.com/xanderfrangos/twinkle-tray/wiki/Display-Detection-&-Support-Issues

ClickMonitorDDC works fine and doesn't show any issue at all. So this is just a twinkle tray problem
